### PR TITLE
Remove references to OAuth and the deprecated Seldon OAuth gateway

### LINKED ISSUE #1677: Remove OAuth code from Seldon Client
The `SeldonClient` Python implementation currently has some code related to handling OAuth tokens. Since this is no longer used (and not much maintained) it would be good to remove this.  

https://github.com/SeldonIO/seldon-core/blob/f79d58c429f0b709aef71cf083341a62bbcc82c5/python/seldon_core/seldon_client.py#L1237-L1276

### PR DESCRIPTION
**Pull request by adriangonz**
**What this PR does / why we need it**:

Remove deprecated OAuth code from `seldon_core` as well as some references to the deprecated `oauth_key` and `oauth_secret` fields of `SeldonDeployment`. 

**Which issue(s) this PR fixes**:

Fixes #1677 

**Special notes for your reviewer**:

This PR tries to remove most of the references to OAuth and the deprecated Seldon gateway but it's possible that some of them still remain.
There is a `seldon_core.SeldonCallCredentials` object which holds an auth token, but that one seems to have a scope different to the OAuth code (and could still be potentially useful).

This PR doesn't remove the `oauth_key` and `oauth_secret` fields from the CRD as that would mean breaking changes within the same version.
So these fields are still there but flagged as deprecated. 

This PR also includes a fix for the `hack/update_python_version.sh` script, which was breaking a couple notebooks (e.g. https://github.com/SeldonIO/seldon-core/commit/9ce35af1ba7dc8cfc1361ef3e806cd5035739ee3#diff-df3b13e162caddc01763ecf53f05bc42a0718582e9f5bd3ebb82a558d914e782).
 
**Does this PR introduce a user-facing change?**:

```release-note
Remove support of the deprecated Seldon OAuth gateway from `seldon_core`.
```

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **adriangonz**
You can assign the PR to them by writing `/assign @adriangonz` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Tue Nov 10 16:00:26 UTC 2020
The logs for [pr-build] [1] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/1.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=1

**Comment by seldondev** [comment]
Tue Nov 10 16:01:05 UTC 2020
The logs for [lint] [2] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/2.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=2

**Comment by seldondev** [comment]
Tue Nov 10 16:03:13 UTC 2020
The logs for [notebooks] [3] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/3.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=3

**Comment by seldondev** [comment]
Tue Nov 10 16:03:32 UTC 2020
The logs for [integration] [4] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/4.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=4

**Comment by seldondev** [comment]
Tue Nov 10 16:05:01 UTC 2020
The logs for [docs] [5] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/5.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=5

**Comment by seldondev** [comment]
@adriangonz: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
docs | 3b01e4e52caf888934e6efc00f104b72576ccb7e | [link]() | `/test docs`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@adriangonz: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
docs | 3b01e4e52caf888934e6efc00f104b72576ccb7e | [link]() | `/test docs`
integration | 3b01e4e52caf888934e6efc00f104b72576ccb7e | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Tue Nov 10 16:24:06 UTC 2020
The logs for [integration] [6] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/6.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=6

**Comment by seldondev** [comment]
@adriangonz: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
docs | 3b01e4e52caf888934e6efc00f104b72576ccb7e | [link]() | `/test docs`
integration | 3b01e4e52caf888934e6efc00f104b72576ccb7e | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@adriangonz: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
docs | 3b01e4e52caf888934e6efc00f104b72576ccb7e | [link]() | `/test docs`
integration | 3b01e4e52caf888934e6efc00f104b72576ccb7e | [link]() | `/test integration`
notebooks | 3b01e4e52caf888934e6efc00f104b72576ccb7e | [link]() | `/test notebooks`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Wed Nov 11 11:44:37 UTC 2020
The logs for [pr-build] [7] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/7.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=7

**Comment by seldondev** [comment]
Wed Nov 11 11:44:39 UTC 2020
The logs for [lint] [8] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/8.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=8

**Comment by seldondev** [comment]
Wed Nov 11 11:45:02 UTC 2020
The logs for [docs] [9] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/9.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=9

**Comment by seldondev** [comment]
Wed Nov 11 12:06:14 UTC 2020
The logs for [integration] [10] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/10.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=10

**Comment by seldondev** [comment]
Wed Nov 11 14:08:38 UTC 2020
The logs for [notebooks] [11] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/11.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=11

File: examples/models/disruption_budgets/pdbs_example.ipynb
Comment (RafalSkolasinski) [inline examples/models/disruption_budgets/pdbs_example.ipynb:26]: I always worry when I see absolute links... One of the issues is that even if someone will be browsing docs for a given version link will lead to latest...

Comment (adriangonz) [reply examples/models/disruption_budgets/pdbs_example.ipynb:26]: I agree. Unfortunately, there isn't a better alternative when using `nbsphinx-link`.

Comment (RafalSkolasinski) [reply examples/models/disruption_budgets/pdbs_example.ipynb:26]: Maybe would be worth moving all examples / notebooks inside `doc/` folder at some point? Would that help?

**Review by RafalSkolasinski — CHANGES_REQUESTED** [review]
Looks good, nice to see code cleanup :+1: 

I think one more place to adjust in in `batch_processor.py`:
```
CHOICES_GATEWAY_TYPE = ["ambassador", "istio", "seldon"]
```
probably need to remove `seldon` gateway type

**Comment by seldondev** [comment]
@adriangonz: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | f0bf07cea631f7f6b4b6359e4a99f29eb2cc7996 | [link]() | `/test notebooks`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: examples/models/disruption_budgets/pdbs_example.ipynb
Comment (adriangonz) [reply examples/models/disruption_budgets/pdbs_example.ipynb:26]: That would help, and that's kind of what nbsphinx favours. However, I believe it was decided not to do that to keep the examples folder at the root level of the repo.

Maybe an option worth exploring would be using sym links? Not sure how that would work with Git and nbsphinx.

Comment (RafalSkolasinski) [reply examples/models/disruption_budgets/pdbs_example.ipynb:26]: Symlinks seems reasonable, imo. Maybe worth following an follow up issue on that.

**Comment by RafalSkolasinski** [comment]
My comments are more of a "nitpick"  - batch_processor adjustment I am happy to take care in follow up.
Unless there is anything else @adriangonz to change here, I would say that we can merge it

File: examples/models/disruption_budgets/pdbs_example.ipynb
Comment (adriangonz) [reply examples/models/disruption_budgets/pdbs_example.ipynb:26]: Good shout. I'll open up an issue.

**Comment by adriangonz** [comment]
@RafalSkolasinski I've kicked off the notebook tests again. Once those pass I think we should be good to merge.

**Review by RafalSkolasinski — APPROVED** [review]
/lgtm
/approve

**Comment by seldondev** [comment]
Mon Nov 16 14:01:51 UTC 2020
The logs for [notebooks] [12] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/12.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=12

File: examples/models/disruption_budgets/pdbs_example.ipynb
Comment (adriangonz) [reply examples/models/disruption_budgets/pdbs_example.ipynb:26]: For reference, this is the issue exploring the symlinks: https://github.com/SeldonIO/seldon-core/issues/2649

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[RafalSkolasinski](https://github.com/SeldonIO/seldon-core/pull/2623#pullrequestreview-531344891 "Approved")*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [RafalSkolasinski]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Mon Nov 16 15:13:18 UTC 2020
The logs for [lint] [14] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/14.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=14

**Comment by seldondev** [comment]
Mon Nov 16 15:13:17 UTC 2020
The logs for [pr-build] [13] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/13.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=13

**Comment by seldondev** [comment]
Mon Nov 16 15:13:21 UTC 2020
The logs for [integration] [17] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/17.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=17

**Comment by seldondev** [comment]
Mon Nov 16 15:13:38 UTC 2020
The logs for [notebooks] [15] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/15.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=15

**Comment by seldondev** [comment]
Mon Nov 16 15:16:02 UTC 2020
The logs for [docs] [16] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/16.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=16

**Comment by seldondev** [comment]
@adriangonz: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | f0bf07cea631f7f6b4b6359e4a99f29eb2cc7996 | [link]() | `/test notebooks`
docs | 70b62f5e8effec9e3a77b323a2fd663db50a3601 | [link]() | `/test docs`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by adriangonz** [comment]
The `docs` linter because of a broken external link. It could be something temporary, so we'll leave it there for now.

```
tutorials/blogs.rst:3: [broken] https://www.mosaicventures.com/mosaicblog/2020/2/20/mlops-the-end-of-end-to-end: 404 Client Error: Not Found for url: https://www.mosaicventures.com/mosaicblog/2020/2/20/mlops-the-end-of-end-to-end
```

**Comment by seldondev** [comment]
@adriangonz: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | f0bf07cea631f7f6b4b6359e4a99f29eb2cc7996 | [link]() | `/test notebooks`
docs | 70b62f5e8effec9e3a77b323a2fd663db50a3601 | [link]() | `/test docs`
integration | 70b62f5e8effec9e3a77b323a2fd663db50a3601 | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Mon Nov 16 16:17:42 UTC 2020
The logs for [integration] [18] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/18.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=18

**Comment by seldondev** [comment]
@adriangonz: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
docs | 70b62f5e8effec9e3a77b323a2fd663db50a3601 | [link]() | `/test docs`
integration | 70b62f5e8effec9e3a77b323a2fd663db50a3601 | [link]() | `/test integration`
notebooks | f0bf07cea631f7f6b4b6359e4a99f29eb2cc7996 | [link]() | `/test notebooks`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@adriangonz: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
docs | 70b62f5e8effec9e3a77b323a2fd663db50a3601 | [link]() | `/test docs`
notebooks | f0bf07cea631f7f6b4b6359e4a99f29eb2cc7996 | [link]() | `/test notebooks`
integration | 70b62f5e8effec9e3a77b323a2fd663db50a3601 | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Mon Nov 16 17:37:19 UTC 2020
The logs for [integration] [19] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/19.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=19

**Comment by seldondev** [comment]
@adriangonz: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
docs | 70b62f5e8effec9e3a77b323a2fd663db50a3601 | [link]() | `/test docs`
notebooks | f0bf07cea631f7f6b4b6359e4a99f29eb2cc7996 | [link]() | `/test notebooks`
integration | 70b62f5e8effec9e3a77b323a2fd663db50a3601 | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@adriangonz: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
docs | 70b62f5e8effec9e3a77b323a2fd663db50a3601 | [link]() | `/test docs`
integration | 70b62f5e8effec9e3a77b323a2fd663db50a3601 | [link]() | `/test integration`
notebooks | 70b62f5e8effec9e3a77b323a2fd663db50a3601 | [link]() | `/test notebooks`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Tue Nov 17 17:37:31 UTC 2020
The logs for [integration] [20] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/20.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=20

**Comment by seldondev** [comment]
@adriangonz: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
docs | 70b62f5e8effec9e3a77b323a2fd663db50a3601 | [link]() | `/test docs`
notebooks | 70b62f5e8effec9e3a77b323a2fd663db50a3601 | [link]() | `/test notebooks`
integration | 70b62f5e8effec9e3a77b323a2fd663db50a3601 | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Wed Nov 18 09:47:27 UTC 2020
The logs for [integration] [21] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/21.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=21

**Comment by seldondev** [comment]
@adriangonz: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
docs | 70b62f5e8effec9e3a77b323a2fd663db50a3601 | [link]() | `/test docs`
notebooks | 70b62f5e8effec9e3a77b323a2fd663db50a3601 | [link]() | `/test notebooks`
integration | 70b62f5e8effec9e3a77b323a2fd663db50a3601 | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Thu Nov 19 09:38:22 UTC 2020
The logs for [integration] [23] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/23.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=23

**Comment by seldondev** [comment]
Thu Nov 19 09:38:43 UTC 2020
The logs for [notebooks] [22] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/22.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=22

**Comment by seldondev** [comment]
@adriangonz: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
docs | 70b62f5e8effec9e3a77b323a2fd663db50a3601 | [link]() | `/test docs`
notebooks | 70b62f5e8effec9e3a77b323a2fd663db50a3601 | [link]() | `/test notebooks`
integration | 70b62f5e8effec9e3a77b323a2fd663db50a3601 | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Thu Nov 19 12:05:22 UTC 2020
The logs for [lint] [25] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/25.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=25

**Comment by seldondev** [comment]
Thu Nov 19 12:05:40 UTC 2020
The logs for [pr-build] [24] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/24.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=24

**Comment by seldondev** [comment]
Thu Nov 19 12:05:52 UTC 2020
The logs for [lint] [27] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/27.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=27

**Comment by seldondev** [comment]
Thu Nov 19 12:05:52 UTC 2020
The logs for [pr-build] [26] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2623/26.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2623 --build=26

**Pull request merged**
